tinyagentos/routes/secrets.py

```
@@ -38,7 +40,25 @@ async def get_agent_secrets(request: Request, agent_name: str):
     return secrets
 
 
-@router.get("/api/secrets/{name}")
+@router.get("/api/secrets/agent/{agent_name}/github")
+async def get_agent_github_grants(
+    request: Request,
+    agent_name: str,
+    user: dict = Depends(get_current_user),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)